testdata/vast4.2_click_through_in_wrapper.xml

```diff
@@ -0,0 +1,90 @@
+<VAST version="4.0"
```

Comment:
Why all versions are set to 4.0 and not 4.2?

Reply:
will fix